### ISSUE TITLE
Testing partition schedules

### ISSUE THREAD

**Issue by bernardocortez** [dagster-io/dagster#5955]
## Use Case

I have built a schedule using [`build_schedule_from_partitioned_job`](https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules#a-schedule-from-a-partitioned-job), which I need to test.
When I test it [like if had been defined using the `@schedule` decorator](https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules#testing-schedules), I get the following erro message:

> dagster.core.errors.DagsterInvalidInvocationError: Only partition schedules created using one of the partition schedule decorators can be directly invoked.

## Ideas of Implementation

There should be a method for testing this type of schedules as well.

---
#### Message from the maintainers:
Excited about this feature? Give it a :thumbsup:. We factor engagement into prioritization.

**Comment by sryza** [comment]
Hey @bernardocortez - I just posted a PR that will enable direct invocation of partitioned config: https://github.com/dagster-io/dagster/pull/5976.

So, for example, you'll be able to do:

```
@daily_partitioned_config(start_date=...)
def my_partitioned_config(start, end):
    return {"a": "b"}

assert my_partitioned_config(some_start_date, some_end_date) == {"a": "b"}
```
Does that satisfy what you need?

**Comment by bernardocortez** [comment]
> daily_partitioned_config

Perfect. Thanks @sryza!
Will it be available on dagster 0.13.12?

**Comment by sryza** [comment]
Alas, it won't make it in until 0.13.13
